persistence/jdbc/src/test/java/org/infinispan/persistence/jdbc/UnitTestDatabaseManager.java

```diff
@@ -28,14 +30,14 @@
    private static final String DB_TYPE = System.getProperty("infinispan.test.jdbc.db", "H2");
    private static final String H2_DRIVER = org.h2.Driver.class.getName();
    private static final String NON_EXISTENT_DRIVER = "non.existent.Driver";
-   private static final DatabaseType dt;
+   private static final org.infinispan.persistence.jdbc.common.DatabaseType dt;
```

Comment:
No need for the FQN.

Reply:
This was removed with the refactor of the two.